api-gateway/src/main/java/gov/va/vro/OpenApiConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package gov.va.vro;
+
+import gov.va.vro.propmodel.Info;
+import gov.va.vro.propmodel.OpenApiProperties;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Contact;
+import io.swagger.v3.oas.models.info.License;
+import io.swagger.v3.oas.models.servers.Server;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@OpenAPIDefinition
+@Configuration
+public class OpenApiConfiguration {
+  @Autowired
+  private final OpenApiProperties openApiProperties = new OpenApiProperties();
```

Comment:
Isn't it a little unusual to initialize `@autowired` member of a class?  Seems to mildly defeat the purpose of dependency injection.

Reply:
This was copied from https://github.com/department-of-veterans-affairs/abd-vro/blob/a6e0c860d257f4c6373b611c74882b9e9b9b4ef9/app/src/main/java/gov/va/vro/config/OpenApiConfiguration.java#L27

I removed `@Autowired` and added `@RequiredArgsConstructor`.